### ISSUE TITLE
Python3 not showing up in version check

### ISSUE THREAD

**Issue by Japarjam** [smartcontractkit/chainlink-mix#23]
In setting up my environment, when I get to checking to make sure python downloaded properly, I enter python3 --version.  I get an error every time- I have uninstalled, reinstalled, making sure it is the right one- used CL, power shell tried in VS 

Nothing seems to work to make sure this step is properly installing.

**Comment by Japarjam** [comment]
I figured it out- in the Chainlink Bootcamp Docs, it only shows the Mac/Linus python3 --version command- this doesn't work for Windows- for Windows, use py -3  --version.

**Comment by PatrickAlphaC** [comment]
!!!

So the exact command you used was `py -3 --version` ?

**Comment by PatrickAlphaC** [comment]
That's.... odd

**Comment by Japarjam** [comment]
Yes- different for Windows vs Mac/ Linux-  great for a noob like me to struggle and persist- learned a lot this way.  

Thanks for the Bootcamp- loving the learning process.

**Comment by Japarjam** [comment]
Spoke too soon- Now I am stuck on getting Brownie loaded.  I will holler if
I get stuck and can't figure it out
